htlcswitch/interceptable_switch.go

```diff
@@ -548,7 +555,7 @@ func (s *InterceptableSwitch) forward(
 		// We are in interceptor-required mode. If this is a new packet, it is
 		// still safe to fail back. The interceptor has never seen this packet
 		// yet. This limits the backlog of htlcs when the interceptor is down.
-		if !isReplay {
+		if !isReplay && s.mode == HtlcInterceptorModeRequired {
```

Comment:
can update comment below "This packet is a replay." since it may not be a replay

Reply:
updated